### ISSUE TITLE
pre-trained mask rcnn model

### ISSUE THREAD

**Issue by anindyaDP** [CharlesShang/FastMaskRCNN#203]
In previous issues, I have seen models trained using 190K epochs ( https://drive.google.com/file/d/0BwVrmnQ5PwycT0p1Y0tLQV9UYUk/view?usp=sharing ) and 2500K epochs ( https://drive.google.com/open?id=0B0J4gcV0gfL4U1NadkllSndKbFk )
there is a model trained using 5000K epochs ( https://www.dropbox.com/s/k0vuldvzsxc18xu/coco_resnet50_model.ckpt-499999.zip?dl=0 ) where the dropbox link no longer works:

Does anyone have access to other pre-trained tf mask rcnn models? Thanks.

**Comment by santhi-2020** [comment]
on which data you have trained and what objects did this model detects
